.github/workflows/vertexai.yml

```diff
@@ -108,10 +139,15 @@ jobs:
           - os: macos-14
             xcode: Xcode_15.2
     runs-on: ${{ matrix.os }}
+    needs: spm-package-resolved
     env:
       FIREBASECI_USE_LATEST_GOOGLEAPPMEASUREMENT: 1
     steps:
     - uses: actions/checkout@v4
+    - uses: actions/cache/restore@v4
+      with:
+        path: .build
+        key: ${{needs.spm-package-resolved.outputs.cache_key}}
     - name: Xcode
       run: sudo xcode-select -s /Applications/${{ matrix.xcode }}.app/Contents/Developer
     - name: Initialize xcodebuild
```

Comment:
Delete `xcodebuild -list`. It's no longer needed now that SPM is initialized

Reply:
nvm, this is the sample

Reply:
I think you're still right that it isn't needed. Still builds successfully without it: https://github.com/firebase/firebase-ios-sdk/actions/runs/10967631419/job/30457712637?pr=13687